spring-cloud-skipper-server/src/test/java/org/springframework/cloud/skipper/repository/PackageMetadataMvcTests.java

```diff
@@ -59,9 +64,28 @@ public void testProjection() throws Exception {
 				.andExpect(jsonPath("$._embedded.packageMetadata[0].iconUrl")
 						.value("http://www.gilligansisle.com/images/a2.gif"))
 				.andExpect(jsonPath("$._embedded.packageMetadata[0].description").doesNotExist())
+				.andExpect(jsonPath("$._embedded.packageMetadata[0]._links.install.href")
+						.value("http://localhost/packageMetadata/3/install"))
```

Comment:
Given the `packageMetadata` also gets loaded from the default skipper home (if it has index files), we could probably use a different `skipperHome` to use to get the package index directory. This way, we can avoid using specific ids here and could use `1` and `2`.